### PR TITLE
Add a few changes:

### PR DESCRIPTION
**Pull request by uzluisf**
* Remove several `render`'s named parameters that served to provide the
  Mustache template with some template variables. Instead, let the user
  supply whatever template variables the template might need.
* Now the Mustache template can also be supplied with partials
  collected from a directory `partials` in the same directory as
  `main.mustache`.
* Instead of supplying a few semantic blocks as HTML meta tags,
  Pod::To::HTML now supplies all of them as regular template
  variables to the template.
* Add a more sensible CSS stylesheet which is available as the
  template `default-style` to the template.
* Update examples/ directory
* Move documentation to doc/ and generate README.md from it.

### CODE REVIEW

**Comment by JJ** [comment]
I'll check it out momentarily...

File: META6.json [hunk 1]
```diff
@@ -2,20 +2,21 @@
     "perl"        : "6.*",
     "name"        : "Pod::To::HTML",
     "version"     : "0.6.2",
-    "author" : "Perl 6",
-    "description" : "Convert Perl 6 Pod to HTML",
+    "author"      : "Raku",
```
[the remark this answers is not in the document]
Comment (Altai-man) [reply META6.json]: +1 for having `auth` field.

**Comment by softmoth** [comment]
>     1. It's 23 files...

That's a feature, not a bug. ☺ Aside from tweaks to documentation, examples & tests, it's 3 files: The module file itself, a CSS template, and an HTML template.

>     2. it probably needs a backend revision based on pod::to::Cached and Documentable

I could be missing something, but I don't think migrating to Documentable would be easier without these changes. It seems orthogonal? If so, I see no reason not to merge this now.

Tim

**Comment by softmoth** [comment]
JJ, you're right that changing the extension in a separate commit would have been cleaner. But thanks for merging the changes in. The end result looks good.

**Pull request merged**